web/main.ts

```diff
@@ -216,6 +180,8 @@ try {
       w.webContents.send('port-message', port);
     });
 
+    tray = new TrayMenu(win);
```

Comment:
We set the value here, but as far as I can tell, this is called purely for side effects?